packages/mui-material/src/InputBase/InputBase.js

```diff
@@ -251,14 +255,10 @@ export const InputBaseInput = styled('input', {
   };
 });
 
-const inputGlobalStyles = (
-  <GlobalStyles
-    styles={{
-      '@keyframes mui-auto-fill': { from: { display: 'block' } },
-      '@keyframes mui-auto-fill-cancel': { from: { display: 'block' } },
-    }}
-  />
-);
+const InputGlobalStyles = globalCss({
```

Comment:
We should document this change in the API in the migration guide, if we are not doing it already.

Reply:
Btw, does this syntax works with styled-components too?